src/app/utils/debug.js

```diff
@@ -15,6 +17,12 @@ const getDebugger = () => {
     };
   }
 
+  if (store.get('debug.disabled')) {
```

Comment:
Hmm, maybe we can solve this differently. Debug has the functionality to check based on env var DEBUG=<prefix> what messages to show. Maybe we can put this in localStorage and derive it from that.

Reply:
So is it possible to disable debug messages by running `DEBUG=*something* yarn start`? And if yes, what should `*something*` be?

Reply:
(removed it for now)

Reply:
We do by default DEBUG=*, but if you omit DEBUG you see no messages. You can for example do DEBUG=cs:compiler:* to see all compiler debug messages.

Reply:
That's strange, because I can see all `cs:` messages by default. Maybe because I'm on Windows? :thinking:

Reply:
I enable it by default 😅  (https://github.com/CompuIves/codesandbox-client/blob/master/src/app/utils/debug.js#L19-L20). We can derive this from the localStorage maybe.

Reply:
I commented out that line, and it still showed all messages. That's why I conditionally (based on localStorage) did `debug.disable()` in the first place.

Reply:
Hmm, okay that's strange. Then their logic changed to enable by default.

Reply:
I'll just disable it manually for now. (and be careful to not commit it! :laughing: )